__tests__/components/editor/ResourceTemplateForm.test.js

```diff
@@ -64,7 +64,7 @@ describe('<ResourceTemplateForm />', () => {
 
   it('renders InputLiteral nested component (b/c we have a property of type "literal")', () => {
     expect(wrapper
-      .find('div.ResourceTemplateForm InputLiteral').length)
+      .find('div.ResourceTemplateForm Connect(InputLiteral)').length)
```

Comment:
Can you explain this to me? why is there no corresponding change to ResourceTemplateForm.jsx? Whi is this only needed din the test?